src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -674,4 +674,17 @@ setx PATH "%PATH%;{0}"
   <data name="CommandPrereleaseOptionDescription" xml:space="preserve">
     <value>Allows prerelease packages to be installed.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SelfContainedOptionDescription" xml:space="preserve">
+    <value>Publish the .NET runtime with your application so the runtime doesn't need to be installed on the target machine.
+The default is 'true' if a runtime identifier is specified.</value>
+  </data>
+  <data name="NoSelfContainedOptionDescription" xml:space="preserve">
```

Comment:
Can you call this `FrameworkDependent` instead of `NoSelfContained`? Framework dependent is the product default so it seems odd to pivot on self-contained. I know this is just a property name, but seems worth some consideration.